src/tiled/changewangsetdata.cpp

```diff
@@ -129,6 +129,43 @@ void ChangeWangSetColorCount::redo()
 }
 
 
+ChangeWangSetFlipping::ChangeWangSetFlipping(TilesetDocument *TilesetDocument,
+                                             WangSet *wangSet,
+                                             ChangeType _which,
+                                             bool newValue)
+    : mTilesetDocument(TilesetDocument), mWangSet(wangSet), mWhich(_which),
```

Comment:
Please remember to call the `QUndoCommand` constructor with the name of the undo action. The name can be generic though, like "Change Wang Set", since I don't think there is much value in being very specific here.